src/dash/student_view/StudentDash.jsx

```diff
@@ -1,180 +1,74 @@
+    }
 
     handleSearchClicked(subject, time) {
-        this.setState({searchType:"searchResults", searchSubject:subject, searchTime:time});
-        this.getDisplayTutors("searchResults", subject, time);
+        this.updateTutors("searchResults", subject, time);
         this.forceUpdate();
     }
 
-    getDisplayTutors(searchType, subject, time) {
-        if (searchType === "online") {
-            var self = this;
-            return axios.get('/api/onlineTutors')
-                .then(function (response) {
-                    if (response.data !== '') {
-                        self.setState({displayTutors:response.data});
-                    } else {
-                        console.log(response.data);
-                    }
-                })
-                .catch(function (error) {
-                    console.log(error);
-                });
-        } else {
-            var self = this;
-            let data = {subject:subject, availability:time};
-            axios.get('/api/onlineTutors', {params:data})
-                .then(function (response) {
-                    if (response.data !== '') {
-                        self.setState({displayTutors:response.data});
-                    } else {
-                        console.log(response.data);
-                    }
-                })
-                .catch(function (error) {
-                    console.log(error);
-                });
+    updateTutors(searchType, subject, time) {
+        let self = this;
+        let data = {};
+        if (searchType !== "online") {
+          data = {subject: subject, availability: time};
         }
+
+        axios.get('/api/onlineTutors', {params: data})
+          .then(function (response) {
+            if (response.data !== '') {
+              self.props.changeTutors(response.data);
+              self.props.changeSearchDisplay(searchType, subject, time);
+            } else {
+              console.log(response.data);
+            }
+          })
+          .catch(function (error) {
+            console.log(error);
+          });
     }
 
     render() {
-
         return (
             <div>
                 <div className="col-md-12 atlanta">
                     <DashSearchBar handleSearchClicked={this.handleSearchClicked}/>
                 </div>
-                <DefaultDashTutorList subject={this.state.searchSubject} time={this.state.searchTime} data={this.state.displayTutors ? this.state.displayTutors : []} searchType={this.state.searchType}/>
+                <TutorSearchList/>
             </div>
         );
     }
 }
 
-export default DefaultDash;
\ No newline at end of file
+const mapStateToProps = (state) => {
+  return state;
+};
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    changeTutors: (tutors) => dispatch(changeTutorsAction(tutors)),
+    changeSearchDisplay: (search_type, search_subject, search_time) => dispatch(onSearchAction(search_type, search_subject, search_time))
```

Comment:
you should look into using ```bindActionCreators(actionCretors, dispatch)```  [get more info here](https://github.com/reactjs/redux/blob/master/docs/api/bindActionCreators.md)